pytorch/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM fedora
+
+RUN dnf install -y \
+        python3.7 \
+        python3-virtualenv.noarch \
+        git \
+        blas-devel \
+        gcc-c++ \
+        cmake \
+        python3-devel \
+        protobuf-compiler \
+        protobuf-devel \
+        llvm7.0-devel \
+        redhat-rpm-config \
+        libGL
+
+RUN cd && \
+    virtualenv -p 3.7 venv && \
+    source venv/bin/activate && \
+    python -m pip install --upgrade pip && \
+    pip install \
+        torch \
+        torchtext \
+        torchvision
+
+RUN cd && \
```

Comment:
```suggestion
RUN cd && \
    source venv/bin/activate && \
```

Reply:
Oops I saw these comments later, but pushed the changes already.